WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -71,6 +67,15 @@ public async Task StartCoinJoinAsync(CancellationToken cancellationToken)
 			// Confirm coins.
 			aliceClients = await ConfirmConnectionsAsync(aliceClients, roundState.ConnectionConfirmationTimeout, cancellationToken).ConfigureAwait(false);
 
+			// Reissuance.
+			var bobClient = CreateBobClient(roundState);
+			var outputAmountsWithCredentials = await ReissuancesAsync(bobClient, cancellationToken).ConfigureAwait(false);
+
+			// Generate and add output addresses.
+			Keymanager.AssertLockedInternalKeysIndexed(howMany: outputAmountsWithCredentials.Count());
+			var allLockedInternalKeys = Keymanager.GetKeys(x => x.IsInternal && x.KeyState == KeyState.Locked);
+			var outputs = outputAmountsWithCredentials.Zip(allLockedInternalKeys, (amount, hdPubKey) => new TxOut(amount.Amount, hdPubKey.P2wpkhScript));
```

Comment:
Although we want all output registrations to mostly follow all reissuance requests in the happy path (and we want this to be true on average for all clients), for the requests of a single client if one of the reissuance requests causes any delay, output registrations that don't depend on it should not be affected